### PR TITLE
Footer_small the icons of contributors

### PR DESCRIPTION
**Pull request by DavidMarcosJ**
Footer added, with contributor icons and social icons.

image

image

### CODE REVIEW

**Review by dhairyagothi — CHANGES_REQUESTED** [review]
image


increase width to full and remove this contributor images

**Comment by DavidMarcosJ** [comment]
Okay, I'll get to it.

File: public/index.html [hunk 1]
```diff
@@ -422,6 +423,47 @@
...
+                <a href="https://github.com/MonishRaman" target="_blank">
+                <p>MonishRaman</p></a>
+                <a href="https://github.com/kom-senapati" target="_blank">
+                <p>Kom-senapati</p></a>
+                <a href="https://github.com/Vansh-kash2023" target="_blank">
+                <p>Vansh-kash2023</p></a>
+                <a href="https://github.com/externref" target="_blank">
+                <p>Externref</p></a>
+                <a href="https://github.com/SuhainaFathimaM" target="_blank">
+                <p>SuhainaFathimaM</p></a>
```
Comment (dhairyagothi) [inline public/index.html]: remove contributor part

**Review by dhairyagothi — CHANGES_REQUESTED** [review]

**Comment by dhairyagothi** [comment]
good @DavidMarcosJ

**Comment by dhairyagothi** [comment]
give email id to assign score

**Comment by DavidMarcosJ** [comment]
davidmarcosj98@gmail.com

**Comment by dhairyagothi** [comment]
Wrong email ID @DavidMarcosJ

**Comment by DavidMarcosJ** [comment]
My mistake, I had the email in hidden, try again. Thanks @dhairyagothi

**Comment by dhairyagothi** [comment]
image


email id not working

**Comment by DavidMarcosJ** [comment]
I assume it is the same email as the github one because I am not registered anywhere called vSoC Manager or something like that.

**Comment by dhairyagothi** [comment]
> I assume it is the same email as the github one because I am not registered anywhere called vSoC Manager or something like that.

are you a participant in Vsoc ?

**Comment by DavidMarcosJ** [comment]
Nop

**Comment by DavidMarcosJ** [comment]
> > I assume it is the same email as the github one because I am not registered anywhere called vSoC Manager or something like that.
> 
> are you a participant in Vsoc ?

No, how can I participate?

**Comment by dhairyagothi** [comment]
> > > I assume it is the same email as the github one because I am not registered anywhere called vSoC Manager or something like that.
> > 
> > 
> > are you a participant in Vsoc ?
> 
> No, how can I participate?

it is over now but thanku for your contribution in my repo

**Pull request merged**